### ISSUE TITLE
Geometry Error Causes VSP to Crash

### ISSUE THREAD

**Issue by EmileeWeir** [OpenVSP/OpenVSP#124]
This just started happening a couple of days ago - I initially thought it was an issue with my external hard drive (I keep dropping it, causing it to unplug while running) and was originally not going to worry about it, but it just now happened to a model I started and saved directly to my laptop's internal hard drive. I need the vertical tail section to be a certain area and am now finding that, as I change the Section Planform geometries and, it will randomly change all of the non-selected planform geometries to "nan(undefined)". When this happens, if I attempt to change any of the Section Planform options for that particular geometry, it crashes the program. It will continue to do this every time I open the model, and eventually, the model cannot be opened. It will then return an error message when I try to open the model: "Failure in SkinRibs set_conditions". I was originally running version 3.16.1-win32 where the issue started. This continues to happen with the latest version. I have no idea why VSP is doing this and it has never happened before, but I suspect it could be related to the latest Windows updates (disclaimer: I am by no means even an amateur with computers but this is the only cause I can think of).

**Comment by EmileeWeir** [comment]
Update: Yesterday, VSPAero also stopped working when trying to calculate models at 0 degrees AOA:

![image](https://user-images.githubusercontent.com/49534820/59274251-f09dd700-8c27-11e9-8eb1-f791db9ceae4.png)

VSPAero is also producing weird results when other angles are run:

![image](https://user-images.githubusercontent.com/49534820/59274692-dc0e0e80-8c28-11e9-87e5-ae5193c87b86.png)
![image](https://user-images.githubusercontent.com/49534820/59274775-06f86280-8c29-11e9-9ac0-9d1cf9bd5fee.png)
![image](https://user-images.githubusercontent.com/49534820/59274797-14155180-8c29-11e9-9f6c-11f35716b669.png)
![image](https://user-images.githubusercontent.com/49534820/59274829-22fc0400-8c29-11e9-9f9f-9744427595ea.png)

I tried having Microsoft Support reinstalling an older version of Microsoft (should have been the update before all of this occurred), as it is also affecting my ability to run Microsoft SQL. VSPAero still will not run models at a 0 degree AOA, even after deleting and reinstalling OpenVSP.

**Comment by EmileeWeir** [comment]
Update: I'm still having issues with it corrupting files, but now I cannot even get VSPAero to run! It crashes every time I try to run the program on a file I just created a few days ago (though strangely it seems to still work with older models). I have tried contacting Microsoft Support about this four times now (the first time my Windows 10 version was restored to a previous version, the second time Windows 10 updated, the third ended up getting disconnected, and the fourth suggested contacting OpenVSP support while they submitted a report to Windows 10 engineers) and nothing thus far has provided a more than temporary fix. I need to be able to run VSPAero on this new file and am hoping that it does not corrupt!

![image](https://user-images.githubusercontent.com/49534820/60818933-96336000-a16c-11e9-99fd-8457bb521261.png)
